examples/check-downloaded-file-name-and-content/README.MD

```diff
@@ -0,0 +1,21 @@
+# Check the Name and the Contents of a Downloaded File
+
+**Test Code**: [index.js](index.js)
+
+This example shows how to check the name and the content of a file downloaded during a test.
+
+During the test, TestCafe clicks the link that triggers file download. To verify download success and obtain the file name and content, a [RequestLogger](https://devexpress.github.io/testcafe/documentation/reference/test-api/requestlogger/) is attached to the test with the [t.requestHooks](https://devexpress.github.io/testcafe/documentation/reference/test-api/test/requesthooks.html) method.
+
+The [RequestLogger.contains](https://devexpress.github.io/testcafe/documentation/reference/test-api/requestlogger/contains.html) method in combination with the [t.expect.ok](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/ok.html) verify the downloaded file information.
```

Comment:
What if we say `name and content`? I don't think file content qualifies as `file information` so we may probably be saying it wrong.

Reply:
Yes, `name and content` is more precise. I edited it in.